app/src/main/res/values-qq/strings.xml

```diff
@@ -1619,4 +1619,27 @@
+  <string name="patroller_saved_message_title_10">Title text for the ready-to-use user talk message offered to experienced editors to warn another editor about article improvement suggestion.</string>
+  <string name="patroller_saved_message_1">Ready-to-use user talk message offered to experienced editors to warn another editor about vandalism. Please do not translate the keywords formatted in triple braces {{{...}}}. Make sure that all links are formatted as wikitext links, pointing to your language wiki. if a certain link doesn\'t exist in your wiki language, try linking to a similar page on Meta wiki.</string>
+  <string name="patroller_saved_message_2">Ready-to-use user talk message offered to experienced editors to warn another editor about editing tests. Please do not translate the keywords formatted in triple braces {{{...}}}. Make sure that all links are formatted as wikitext links, pointing to your language wiki. if a certain link doesn\'t exist in your wiki language, try linking to a similar page on Meta wiki.</string>
+  <string name="patroller_saved_message_3">Ready-to-use user talk message offered to experienced editors to warn another editor about a possible violation of wikipedia\'s neutral point of view. Please do not translate the keywords formatted in triple braces {{{...}}}. Make sure that all links are formatted as wikitext links, pointing to your language wiki. if a certain link doesn\'t exist in your wiki language, try linking to a similar page on Meta wiki.</string>
+  <string name="patroller_saved_message_4">Ready-to-use user talk message offered to experienced editors to warn another editor about a possible poor quality machine translation. Please do not translate the keywords formatted in triple braces {{{...}}}. Make sure that all links are formatted as wikitext links, pointing to your language wiki. if a certain link doesn\'t exist in your wiki language, try linking to a similar page on Meta wiki.</string>
+  <string name="patroller_saved_message_5">Ready-to-use user talk message offered to experienced editors to warn another editor about conflict of interest. Please do not translate the keywords formatted in triple braces {{{...}}}. Make sure that all links are formatted as wikitext links, pointing to your language wiki. if a certain link doesn\'t exist in your wiki language, try linking to a similar page on Meta wiki.</string>
+  <string name="patroller_saved_message_6">Ready-to-use user talk message offered to experienced editors to offer a final warning to another editor. This will be used in case where a user has made multiple destructive edits despite multiple warnings.Please do not translate the keywords formatted in triple braces {{{...}}}. Make sure that all links are formatted as wikitext links, pointing to your language wiki. if a certain link doesn\'t exist in your wiki language, try linking to a similar page on Meta wiki.</string>
```

Comment:
For messages that don't contain any links, is it necessary to tell the translators about formatting links? This might be a bit confusing.